draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -253,10 +197,9 @@ verification is not observable.
 
 # Intermediary Considerations {#intermediary}
 
-Since the Signature and HMAC HTTP Authentication Schemes leverage TLS keying
-material exporters, their output cannot be transparently forwarded by HTTP
-intermediaries. HTTP intermediaries that support this specification have two
-options:
+Since the Signature HTTP authentication scheme leverages TLS keying material
+exporters, its output cannot be transparently forwarded by HTTP intermediaries.
+HTTP intermediaries that support this specification have two options:
 
 * The intermediary can validate the authentication received from the client,
 then inform the upstream HTTP server of the presence of valid authentication.
```

Comment:
I just noticed below that both of these options depend on the intermediary being trusted to represent the status or value faithfully.  That makes the second option far less appealing, unless the intermediary doesn't want to perform the validation (but an intermediary is probably in a better position to cache the per-connection status so that extra work can be avoided, so...).

Reply:
Yeah, the number of dragons therein is what prompted me to punt this out of scope. To be fair since the intermediary has the TLS cert private key, this isn't the worst thing it can do.

Reply:
Maybe open an issue and make that assumption explicit.

Reply:
Agreed, done via #2576